frontend/institution/base_institution_page.html

```diff
@@ -83,7 +83,9 @@
                                             Seguidores
                                         </md-button>
                                     </md-menu-item>
-                                    <md-menu-item>
+                                    <!--TODO: Show the item below when hierarchical invitations can be sent 
+                                    @author: Mayza Nunes 10/01/17-->
+                                    <md-menu-item ng-if="false">
```

Comment:
To improve security, we should protect the handlers from direct access. An option it creates an exception that throws an error when a function it's disabled.

Reply:
The handler who send invite for hierarchy institution is the same used to send invite for institutions without hierarchy, so it's not possible throws an error when the post is called.